pull/github.go

```diff
@@ -260,6 +267,28 @@ func (ghc *GitHubContext) Reviews() ([]*Review, error) {
 	return ghc.reviews, nil
 }
 
+func (ghc *GitHubContext) ListRepositoryCollaborators() (map[string]string, error) {
+	if ghc.collaborators == nil {
+		if err := ghc.loadCollaboratorData(); err != nil {
```

Comment:
style: your call on this, but I probably would have inlined this here, since that's the only place it is called. `loadPagedData` is called in multiple places, while `loadCommits` was broken out because it is only part of the operation.